edx_proctoring/templates/proctoring/seq_proctored_exam_instructions.html

```diff
@@ -0,0 +1,125 @@
+{% load i18n %}
+<div class="sequence proctored-exam instructions" data-exam-id="{{exam_id}}" data-exam-started-poll-url="{{exam_started_poll_url}}">
+  <h3>
+    {% blocktrans %}
+      Awaiting Proctoring Installation & Set Up
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      You've chosen to take {{display_name}} as a proctored exam. You should be directed to a new window
+      with installation and setup instructions. You can also <a href="{{software_download_url}}" target="_blank">open the installation window directly.</a>
+    {% endblocktrans %}
+  </p>
+  <div class="proctored-exam-message">
+    <h3>
+    {% blocktrans %}
+      Here is your unique exam code. You'll be asked for it during the setup.
+    {% endblocktrans %}
+    </h3>
+    <h2> {{exam_code}}<span class='copy-to-clipboard'></span></h2>
+    <p>
+      {% blocktrans %}
+        Please do not share this code. It can only be used once and it tied to your {{platform_name}} account.
```

Comment:
Strike "Please": "Do not share this code. It is linked to your {{edX}} account and can be used only once."